runtime/src/bank/tests.rs

```diff
@@ -6527,26 +6467,27 @@ fn test_bank_hash_consistency() {
         if bank.slot == 0 {
             assert_eq!(
                 bank.hash().to_string(),
-                "Hn2FoJuoFWXVFVnwcQ6peuT24mUPmhDtXHXVjKD7M4yP",
```

Comment:
Before: bank0 has blockhash target_per_lamports = 0, and other blockhash has 5000;
Now: all bank has lamports_per_signature = 5000. 
Hence changed bank hash. they are still consistent.